tox.ini

```diff
@@ -38,6 +38,7 @@ find_links =
 
 [testenv]
 deps =
+    -rrequirements.txt
```

Comment:
`requirements.txt`

Reply:
@hughhhh that's not a typo. Per the [documentation](http://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints) there's no space between `-r` and `requirements.txt, I presume because the split the `deps` list on whitespace.

Reply:
Gotcha!